packages/dcos-history/extra/history/server.py

```diff
@@ -78,16 +80,15 @@ def _buffer_response_(name):
 def _response_(content):
     return Response(response=content, content_type="application/json", headers=headers_cb())
 
-
-def start():
+if __name__ == '__main__':
```

Comment:
In the systemd unit you make a change so that gunicorn executable is the main thread and calls out to the app WSGI object. If that't the way to run it, then this whole section should be removed so that we only have one way to run this service